### PR TITLE
Add scenarios for MVC benchmark apps

### PR DESCRIPTION
**Pull request by dougbu**
- part of aspnet/Mvc#7805

### CODE REVIEW

File: build/scenarios.sh [hunk 1]
```diff
@@ -148,6 +150,18 @@ jobs=(
...
+  "--scenario BasicApi.GetUsingRouteValueWithoutToken $trend $basicApiJobs"
+  "--scenario BasicApi.GetUsingRouteValueWithoutAuthorization $trend $basicApiJobs"
+  "--scenario BasicApi.GetUsingRouteValue $trend $basicApiJobs"
+  "--scenario BasicApi.Post $trend $basicApiJobs"
+
+  # BasicViews
+  "--scenario BasicViews.GetTagHelpers $trend $basicViewsJobs"
+  "--scenario BasicViews.PostWithoutToken $trend $basicViewsJobs"
+  "--scenario BasicViews.PostIgnoringToken $trend $basicViewsJobs"
+  "--scenario BasicViews.Post $trend $basicViewsJobs"
```
Comment (rynowak) [inline build/scenarios.sh]: Do we really need to track all of these things? I imaged that a single Get/Post for each app would be enough. I think it's great that we can **measure** all of the various increments, but I'm no sure how much value we get out of tracking them all.

Comment (dougbu) [reply build/scenarios.sh]: Ah, I misunderstood your comments in aspnet/Mvc@7944 and thought you wanted to track the Get and Post variants over time as well. If we focus on the "full boat" scenarios, the list would be:
- BasicApi.GetUsingRouteValue
- BasicApi.Post
- BasicViews.GetTagHelpers
- BasicViews.Post

Thougths?

Comment (rynowak) [reply build/scenarios.sh]: Sounds great!

**Comment by dougbu** [comment]
86dc85338c

**Pull request closed without merging**